Adyen/UI/Form/Items/Address/FormFullAddressItem.swift

```diff
@@ -166,9 +177,32 @@ public final class FormAddressItem: FormValueItem<PostalAddress, AddressStyle>,
         case .stateOrProvince:
             observers[.stateOrProvince] = bind(item.publisher, to: self, at: \.value.stateOrProvince)
             publisherObservers[.stateOrProvince] = observe(publisher) { item.value = $0.stateOrProvince ?? "" }
+        case .country:
+            break
+        }
+    }
+    
+    private func updateItems(identifier: String?) {
+        let items = items + [headerItem, countrySelectItem]
+        for item in items {
+            if let item = item as? FormSplitItem {
+                update(item: item.leftItem, identifier: identifier)
+                update(item: item.rightItem, identifier: identifier)
+                continue
+            }
+            update(item: item, identifier: identifier)
         }
     }
     
+    private func update(item: FormItem, identifier: String?) {
```

Comment:
Why do we need to go thought every item and check if it identifier contains class name?
You can simply assign identifier to SplitItem and then recalculate identifiers for subitems via ViewIdentifierBuilder

Reply:
Check my answer to @mohammedDehairy's comment